More bootstrap4 conversion

bcholmes opened the pull request:
This pull request converts more pages to use Bootstrap 4:

Participant view:

- General Interests
- My Suggestions
- My Schedule

Staff View

- Reports In Category
- View Session Count

olszowka reviewed (changes requested):
Don't like vast sea of white caused by putting entire page in a card, especially on 

-    Staff reports in category
-    Participant general interests

On Participant general interests, the list of roles used to render in three columns, but that
is now broken.

On Staff reports in category, I like the increased spacing, but I'd prefer there'd be clear links or buttons to click rather than the vague rectangles with the hover effect.

bcholmes commented:
> Don't like vast sea of white caused by putting entire page in a card, especially on
> 
>     * Staff reports in category
>     * Participant general interests

Okay. I've removed the cards from these pages. I *do* think that the text tends to complete with the hexagon pattern, but I'll defer to your preference.

> On Participant general interests, the list of roles used to render in three columns, but that
> is now broken.

I've addressed this in ebe8611 

> On Staff reports in category, I like the increased spacing, but I'd prefer there'd be clear links or buttons to click rather than the vague rectangles with the hover effect.

I've addressed this in ebe8611

olszowka commented:
Again, I couldn't figure out how to push a change to this PR so I'm closing it and merging another PR which includes the changes in this one and some tweaks I made.
